code/game/gamemodes/cult/ritual.dm

```diff
@@ -146,11 +149,25 @@ var/engwords = list("travel", "blood", "join", "hell", "destroy", "technology",
 	for(var/mob/living/silicon/ai/AI in player_list)
 		AI.client.images += blood
 
-/obj/effect/rune/examine(mob/user)
-	..()
-	if(iscultist(user))
-		user << "This spell circle reads: <i>[word1] [word2] [word3]</i>."
+/obj/effect/rune/examine()
+	set src in view(2)
+
+	if(!iscultist(usr))
+		usr << "A strange collection of symbols drawn in blood."
+		return
+		/* Explosions... really?
+		if(desc && !usr.stat)
+			usr << "It reads: <i>[desc]</i>."
+			sleep(30)
+			explosion(src.loc, 0, 2, 5, 5)
+			qdel(src)
+		*/
+	if(!desc)
+		usr << "A spell circle drawn in blood. It reads: <i>[word1] [word2] [word3]</i>."
+	else
+		usr << "Explosive Runes inscription in blood. It reads: <i>[desc]</i>."

```

Comment:
Not my doing, that is from how the code was when I started this. I'll fix that though. I've no idea whose idea it was to add explosions however.